coalib/bears/PackageRequirement.py

```diff
@@ -19,3 +19,24 @@ def check(self):
         Check if the requirement is satisfied.
         """
         raise NotImplementedError
+
+
+class PythonRequirement(PackageRequirement):
+
+    def __init__(self, package, version = ""):
+        PackageRequirement.__init__(self, 'pip', package, version)
+
+    @classmethod
+    def multiple(cls, *args):
+        reqs = ()
+        for requirement in args:
+            if isinstance(requirement, str):
+                reqs += cls(requirement)
+            elif isinstance(requirement, tuple):
+                if len(requirement == 2):
+                    reqs += cls(requirement[0], requirement[1])
+                else
+                    raise NotImplementedError
```

Comment:
I would use `TypeError` here too, `NotImplementedError` is for not implemented functions.
or if you want `ValueError`, but if someone wants to catch them he needs two `except` clauses inside a `try`^^
+ provide a message which states what exactly went wrong ("invalid numbers of args provided in tuple" or so)